src/linear_algebra/matrix/nonsingular_inverse.lean

```diff
@@ -373,9 +373,81 @@ begin
         ... = 1 : mul_nonsing_inv B h',
 end
 
-lemma nonsing_inv_right_left (B : matrix n n α) (h : B ⬝ A = 1) : A ⬝ B = 1 :=
+lemma nonsing_inv_right_left (h : B ⬝ A = 1) : A ⬝ B = 1 :=
 B.nonsing_inv_left_right A h
 
+/-- `det A` is a unit implies A is invertible -/
+lemma is_unit_det_of_invertible [invertible A] : is_unit A.det :=
+by apply is_unit_det_of_left_inverse A (invertible.inv_of A) (inv_of_mul_self A)
+
+@[simp,norm]
```

Comment:
What is the `norm` attribute?

Reply:
I don't fully understand `norm` actually. Perhaps delete it?